cmd/lotus-provider/config_new.go

```diff
@@ -0,0 +1,159 @@
+package main
+
+import (
+	"bytes"
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"io"
+
+	"github.com/BurntSushi/toml"
+	"github.com/fatih/color"
+	"github.com/samber/lo"
+	"github.com/urfave/cli/v2"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-address"
+
+	"github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/chain/types"
+	cliutil "github.com/filecoin-project/lotus/cli/util"
+	"github.com/filecoin-project/lotus/cmd/lotus-provider/deps"
+	"github.com/filecoin-project/lotus/node/config"
+	"github.com/filecoin-project/lotus/node/repo"
+)
+
+var configNewCmd = &cli.Command{
+	Name:      "new-cluster",
+	Usage:     "Create new coniguration for a new cluster",
```

Comment:
Right now the command to do that is `lotus-shed miner create`, I do want to create a separate tool for managing the miner actor, so then it will become `sptool create`

The commands that will live in `sptool` (vs `curio` / `lotus-provider`) would be the ones which interact with the miner actor, so things like `create`, managing balances, owner/control addresses, and other on-chain actions could be grouped in that binary.

(I just don't want to do all this in this PR, it's way too big already)

Reply:
Lets refer to an open bug here with these goals in mind, that way it has a continue story for watchers.